docs/native-numbers.md

```diff
@@ -60,14 +60,10 @@ fibonacci = λn // n is the argument
   match n {
     // If the number is 0, then return 0
     0: 0
-    // If the number is greater than 0, bind it predecessor to `a`
-    +a:
-    match a {
-      // If the predecessor is 0, then return 1
-      0: 1
-      // Otherwise, bind n-2 to `b` and return the sum of (fib n-1) and (fib n-2)
-      +b: (+ (fibonacci a) (fibonacci b))
-    }
+    // If the number is 1, then return 1
+    1: 1
+    // Otherwise, and return the sum of (fib (n-2 + 1)) and (fib n-2)
+    2+: (+ (fibonacci (+ n-2 1)) (fibonacci n-2))
```

Comment:
Are the names like `n-2` still binded automatically? If so, should probably still say it here. (unless another doc is saying that and i didn't find)

Reply:
If the successor pattern is `3+` then the bind will be `var-3` and so on

Reply:
Yes, i know, but a doc should be saying that. The name `n-2` is just appearing magically without explanation